src/graphing/radar.js

```diff
@@ -828,12 +828,30 @@ const Radar = function (size, radar) {
     })
 
     if (featureToggles.UIRefresh2022) {
-      renderRadarLegends(radarElement)
+      let hasMovements = hasMovementData(quadrants)
+      renderRadarLegends(radarElement, hasMovements)
```

Comment:
Consolidate into:
`renderRadarLegends(radarElement, hasMovementData(quadrants))` since we don't use the `hasMovements` var again

Reply:
Deal 🤝